tests/test_daemon.cpp

```diff
@@ -83,6 +86,8 @@ void PrintTo(const YAML::Node& node, ::std::ostream* os)
 
 namespace
 {
+const qint64 default_total_bytes{16106127360};
```

Comment:
```suggestion
const qint64 default_total_bytes{16'106'127'360}; // 16G
```

Reply:
Oh, cool, I didn't know one could add `'` to make the number more readable.

Reply:
Yeah introduced in C++14